chainntnfs/neutrinonotify/neutrino.go

```diff
@@ -527,6 +481,79 @@ chainScan:
 	return false
 }
 
+// handleBlocksConnected applies a chain update for a new block. Any watched
+// transactions included this block will processed to either send notifications
+// now or after numConfirmations confs.
+func (n *NeutrinoNotifier) handleBlockConnected(newBlock *filteredBlock) error {
+	// First we'll notify any subscribed clients of the
```

Comment:
The comments (and also the code itself) can be re-flowed now to take advantage of the increased column limit due to the elimination of the nesting.